### ISSUE TITLE
Invalid repository string: http://download.virtualbox.org/virtualbox/debian

### ISSUE THREAD

**Issue by stephansarver** [FGtatsuro/ansible-virtualbox#26]
I am using this role defined as follows in a playbook:

``` 
- { role: FGtatsuro.virtualbox,
        virtualbox_apt_package: 'virtualbox-5.0',
        virtualbox_apt_repository: 'http://download.virtualbox.org/virtualbox/debian',
        virtualbox_allow_unauthenticated: no
      }
```

As described here: https://github.com/FGtatsuro/ansible-virtualbox

I am receiving the following error:

```
fatal: [xxx.xxx.xxx.xxx]: FAILED! => {
    "changed": false, 
    "failed": true, 
    "invocation": {
        "module_args": {
            "filename": null, 
            "install_python_apt": true, 
            "mode": null, 
            "repo": "http://download.virtualbox.org/virtualbox/debian", 
            "state": "present", 
            "update_cache": true, 
            "validate_certs": true
        }, 
        "module_name": "apt_repository"
    }, 
    "msg": "Invalid repository string: http://download.virtualbox.org/virtualbox/debian"
}
```

Please let me know if there is anything else I can provide to help troubleshoot this.

**Comment by stephansarver** [comment]
After further review this is a problem with how I used the 'virtual_box_apt_repository' variable.

To help clarify what I did wrong and looking at the source code here: https://github.com/FGtatsuro/ansible-virtualbox/blob/2728c727e5de3d31f3eee30928b5bd41ecd2fb45/defaults/main.yml

I was able to determine that 'virtual_box_apt_repository' is a multipart space separated string which is better understood from the ansible documentation here: http://docs.ansible.com/ansible/apt_repository_module.html

I think where I got confused here was how this looks in the documentation:

deb http://download.virtualbox.org/virtualbox/debian jessie contrib

vs.

deb 
http://download.virtualbox.org/virtualbox/debian 
jessie contrib


I added notes in the event this is not clear for someone following me and this issue can be closed.
